data/human/human missions.txt

```diff
@@ -6893,3 +6895,387 @@ mission "Lost Dog: Homeward Bound"
+	source "Hippocrates"
+	destination "Maker"
+	to offer
+		has "Adelita 3: done"
+
+	on offer
+		conversation
+			branch "player has a fleet"
+				"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1
+
+			`It's not hard to find a remote canyon within range of the target area. Being so close to the surface of <origin> you're forced to deactivate your shields, but you keep the <ship> hovering in case you need to make a quick escape.`
+				goto release
+
+			label "player has a fleet"
+			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. Being so close to the surface of <origin> you're forced to deactivate your shields, but you keep the <ship> hovering in case you need to make a quick escape.`
+
+			label release
+			`	After settling on a secure location within the canyon, Bunker releases the drones from the open cargo bay; the drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
+
+			`	As you watch the display, the drones arrive on site, and you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
+			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`
+			choice
+				`	"I think you're right."`
+				`	"What about doing both?"`
+					goto plan
+
+			`	"And you know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
+				goto showtime
+
+			label plan
+			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we're trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
+
+			label showtime
+			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, lands near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another - hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
```

Comment:
Why name the Quicksilvers but not what ship the capital ship is?

Reply:
Because I didn't think about that. ~~Blame the person who originally wrote this.~~

Reply:
Done. I couldn't think of good names so I just picked names from Tintin.

Reply:
I meant name what type of ship the capital ship is. We say that you see "a small flotilla of _Quicksilver_" and "a lone Syndicate-built _freighter_" (ends up specifically being a Bulk Freighter, but it's not like we have a lot of options for what that could be), but then this text just says "a capital ship," which could be one of many options.

Reply:
```suggestion
			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a Vanguard flying pirate colors, accompanied by a small flotilla of Quicksilver escorts, lands near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another - hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
```

Reply:
Done. I did mention later that a Vanguard was chasing you into orbit but I've also put that mention here.